tests/mcs_benchmark_data_test/cli/create_benchmark_pipeline_test.py

```diff
@@ -14,20 +14,13 @@
 def test_create_benchmark_pipeline(tmpdir):
 
     TestArgs = NamedTuple(
-        "TestArgs", [("benchmark_name", str), ("using_test_data", bool)]
+        "TestArgs",
+        [("benchmark_name", str), ("using_test_data", bool), ("root_path", Path)],
     )
 
-    test_args = TestArgs("snazzy_new_benchmark", True)
+    test_args = TestArgs("snazzy_new_benchmark", True, tmpdir)
 
-    CreateBenchmarkPipelineCommand()._make_benchmark_directories(
-        root_path=tmpdir, benchmark_name=test_args.benchmark_name
-    )
-
-    CreateBenchmarkPipelineCommand()._create_files_from_template(
-        root_path=tmpdir,
-        benchmark_name=test_args.benchmark_name,
-        data_dir="TEST_DATA_DIR_PATH" if test_args.using_test_data else "DATA_DIR_PATH",
-    )
+    CreateBenchmarkPipelineCommand(args=test_args)()

```

Comment:
If you do
`CreateBenchmarkPipelineCommand(benchmark_name=...)`
You no longer need `TestArgs`.

That said, I think it might help to test a wider swath of code here.
Something like:
```
class Cli:
    def main(self, args=None):
       if args is None:
           # set up the arg parser and parse_args to get args
      else:
           # assume args has the same interface/contract to what you'd get out of parse_args, but has been supplied for testing
      ... rest of code that uses args
```
Then you could do  Cli.main(args=test_args) and it would test the CLI code too.